cmd/up/up.go

```diff
@@ -789,7 +791,12 @@ func (up *upContext) waitUntilExitOrInterruptOrApply(ctx context.Context) error
 		case err := <-up.GlobalForwarderStatus:
 			oktetoLog.Infof("exiting by error in global forward checker: %v", err)
 			return err
-
+		case <-ttySignals:
+			oktetoLog.Infof("SIGTTOU/SIGTTIN received, starting shutdown sequence preventing it to background the process")
+			return oktetoErrors.UserError{
+				E:    fmt.Errorf("terminal disconnect detected"),
```

Comment:
```suggestion
				E:    fmt.Errorf("connection lost to your development container"),
```

We could reuse the current error message that we display when the connection is lost.